test/LaunchDarkly.OpenFeature.ServerProvider.Tests/LaunchDarkly.OpenFeature.ServerProvider.Tests.csproj

```diff
@@ -32,7 +32,7 @@
         <PackageReference Include="LaunchDarkly.ServerSdk" Version="6.3.2" />
         <PackageReference Include="LaunchDarkly.TestHelpers" Version="1.3.0" />
         <PackageReference Include="Moq" Version="4.8.1" />
-        <PackageReference Include="OpenFeature" Version="0.5.0" />
+        <PackageReference Include="OpenFeature" Version="[1.0.0, 2.0.0)" />
```

Comment:
However... I think for the _test_ project, my earlier comment about the wildcard may be applicable. That is, I think we probably want our tests to run against the latest compatible version.

Reply:
Ok. Updated.